partner_address_ok/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from . import test_partner_contact_id
```

Comment:
Hi, I don't think this is the correct test you're trying to import, please fix this.

Reply:
Thank you for finding this error. I have fixed it now.